roboteam_ai/src/skills/Dribble.cpp

```diff
@@ -106,14 +92,19 @@ Dribble::Status Dribble::onUpdate() {
     else if (currentProgress == Progression::WAITING) {
         return Status::Waiting;
     }
-    if(ball->visible) {
-        deltaPos = targetPos - Vector2(ball->pos);
-    }
-    else{
-        deltaPos=targetPos-(Vector2(robot->pos)+Vector2(Constants::ROBOT_RADIUS()+Constants::BALL_RADIUS(),0).rotate(robot->angle));
-    }
+    deltaPos = targetPos - Vector2(ball->pos);
     currentProgress = checkProgression();
 
+    if (currentProgress == STOPPING) {
```

Comment:
what exactly is the point of the new stopping state if all it does is exactly the same as the stopped state?

Reply:
It is used if you want to stop the robot for a few ticks after dribbling

Reply:
Yes, but that's what the stopping state is exactly for <:. This is duplicate functionality